### ISSUE TITLE
Ermöglichung einer alphabetischen Sortierung im Browsing-Modus

### ISSUE THREAD

**Issue by thefass** [ubtue/tuefind#908]
_Die Anregung zu diesem Issue kommt von einem Fachkollegen aus Wuppertal_

Die Ergebnisse, die im Browsing-Modus in Spalten angezeigt werden, sind nach Häufigkeit sortiert. Sie sollen zukünftig auch alphabetisch sortierbar sein. 
Dasselbe Problem hatten wir schon in den Facetten. Dort wurde es gelöst. Vielleicht kann derselbe Lösungsansatz gewählt werden.

**Beispiel_1**
![Bildschirmfoto 2019-10-10 um 15 15 17](https://user-images.githubusercontent.com/14169098/66572225-e1e11580-eb70-11e9-9da0-f137c2544147.png)
Hier müssen die beiden letzten Spalten alphabetisch sortierbar sein

**Beispiel_2**
![Bildschirmfoto 2019-10-10 um 15 18 33](https://user-images.githubusercontent.com/14169098/66572447-47cd9d00-eb71-11e9-8662-6700e27c93fb.png)
Hier muss nur die letzte Spalte alphabetisch sortierbar sein

**Fazit**: immer dann, wenn im Browsingmodus eine Sortierung nach Häufigkeit vorgenommen wird, muss auch eine alphabetische Sortierung ermöglicht werden

**Beispiel der Ermöglichung von Häufigkeits- und alphabetischer Suche in den Facetten:**
![Bildschirmfoto 2019-10-10 um 15 19 31](https://user-images.githubusercontent.com/14169098/66572529-6a5fb600-eb71-11e9-8fe0-9ddb67825c45.png)

**Comment by mtrojan-ub** [comment]
Grundsätzlich steht an der Stelle schon im Hintergrund eine Funktion zur Verfügung mit der man bestimmte Unterkategorien alphabetisch sortieren könnte.

Das Problem ergibt sich allerdings aus dem Handling. Bei bestimmten Feldern mit begrenzter Anzahl (z.B. IxTheo Klassifikation 1. Buchstabe) ist so etwas gut darstellbar, aber wenn man z.B. daran denkt ein Feld wie Verlag oder auch Themen / Schlagwörter anzuzeigen kann es hier ja hunderte bis tausende von Einträgen geben. Aktuell wird wohl auch bei der Sortierung nach Häufigkeit nach einer bestimmten Stelle abgeschnitten, d.h. man bräuchte auch mechanismen zum Suchen / Blättern und so weiter.

So etwas gibt es ja eigentlich auch schon bei Facetten ("mehr"), wobei ich es auch dort schon bei großen Facetten wie z.B. "Verlag" für unschön halte wenn man z.B. alphabetisch sortiert und einen bestimmten Verlag sucht. Da muss man immer wieder auf "mehr" klicken und scrollt sich zu Tode. 

Man sollte also eigentlich eine Art Navigation haben bei der man zumindest auf die Anfangsbuchstaben des Alphabets klicken kann und dann von A auch mal direkt zu S springen kann, wobei das wieder für Sprachen wie Chinesisch, Russisch, Griechisch problematisch sein könnte....

Oder man stellt einen Suchfeld zur Verfügung in das man die ersten X Buchstaben eintippt und der dann die Trefferliste dynamisch filtert. Das wäre denke ich eine bessere Lösung.

So oder so denke ich aber dass das evtl ein Thema für die VuFind Community sein könnte und werde dort mal nachfragen.

**Comment by thefass** [comment]
Ja, Du hast völlig Recht! Ein solcher Einsprung ins Alphabet wäre toll (was habe ich mich schon zu Tode gescrollt) oder eine Möglichkeit die ersten x Buchstaben einzutippen. Danke für die Nachfrage. Würde das Browsing wirklich verbessern!

**Comment by mtrojan-ub** [comment]
Diskussion läuft, siehe https://sourceforge.net/p/vufind/mailman/message/37863619/

**Comment by mtrojan-ub** [comment]
Ich habe zwischenzeitlich eine kurze erste Demo einem Community Call vorgestellt, Demian hat ein ca. 2min langes Video aufgenommen: https://vufind.org/video/facet-demo-20230711.mp4

Resonanz war positiv, in der Zwischenzeit ist ein erster [Pull Request](https://github.com/vufind-org/vufind/pull/2991) an vufind-org offen. Im Optimalfall wird die Änderung in VuFind 9.1 einfließen.
